Added Important Note #5295 (#5504)
MicrosoftDocs/windows-itpro-docs: windows/security/identity-protection/hello-for-business/hello-cert-trust-adfs.md

You need to verify the AD FS service has properly enrolled for an enrollment agent certificate template.  You can verify this is a variety ways, depending on if your service account is a normal user account or if the service account is a group managed service account.

> [!IMPORTANT]

> > if after following the previous steps you are unable to validate that the devices are, in fact, being registrered automatically, there is a group Policy at:

Reviewer: This sentence is cumbersome to read in its current phrasing and layout. I will make a suggestion, but JohanFreelancer9 might have a better suggestion.
```suggestion
> After following the previous steps, if you are unable to validate that the devices are, in fact, being registered automatically, there is a Group Policy at:
```

> > Computer Configuration -> Policies -> Administrative Templates -> Windows Components -> Device Registration -> "Register Domain Joined Computers As Devices", set it to enabled.

Reviewer: From what I have observed previously on this page and earlier, the Microsoft Docs codestyle standard for this type of instruction is more like the following:
```suggestion
> **Computer Configuration > Policies > Administrative Templates > Windows Components > Device Registration >** "Register Domain Joined Computers As Devices". Set the policy to **Enabled**.
```
Worth noting: The dash character ( `-` ) used to create an ASCII arrow symbol will rarely stay aligned to point to the center of the `>` character in HTML text, when displayed on a web page using non-fixed fonts.

> > and the registration will happens automatically.

Reviewer: grammar:
```suggestion
> and the registration will happen automatically.
```
(it is possible that the full sentence structure could be improved, but I don't have a better suggestion at the moment)
